api/src/pcapi/core/offers/api.py

```diff
@@ -1296,3 +1298,33 @@ def delete_price_category(offer: models.Offer, price_category: models.PriceCateg
     validation.check_price_categories_deletable(offer)
     db.session.delete(price_category)
     db.session.commit()
+
+
+def list_nearest_venues_for_offer(
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** proposition de wording: `get_nearest_venues_for_offer` ?